vm/src/obj/objsequence.rs

```diff
@@ -418,3 +418,33 @@ pub fn is_valid_slice_arg(
         Ok(None)
     }
 }
+
+pub fn opt_len(obj: &PyObjectRef, vm: &VirtualMachine) -> Option<PyResult<usize>> {
```

Comment:
Maybe this should be in `VirutalMachine`?

Reply:
I originally had it there, but I called it `get_len` which seemed like a weird name (does it get the length of the `VirtualMachine` itself?), and it seemed slightly niche to be on `VirtualMachine`. Idk, it relates to the sequence protocol, so I figured it would best fit in `objsequence`.